lib/releaselib.sh

```diff
@@ -751,14 +751,28 @@ release::docker::release () {
   # Activate credentials for the k8s.production.user@gmail.com
   [[ "$registry" == "gcr.io/google_containers" ]] \
    && logrun $GCLOUD config set account k8s.production.user@gmail.com
+
+  if [[ -f $(which manifest-tool) ]]; then
+    MANIFEST_TOOL=$(which manifest-tool)
+  else
+    curl -sSL https://github.com/luxas/manifest-tool/releases/download/v0.3.0/manifest-tool > ./manifest-tool
```

Comment:
this step should probably go in `anago`'s `check_prerequisites` function.

and probably rather than automatically downloading, it should present instructions on how to download it.

as-is it'll make the git tree dirty.